R/prepare_predictions.R

```diff
@@ -764,6 +773,145 @@ prepare_predictions_ac <- function(bterms, draws, sdata, oos = NULL,
       }
     }
   }
+  if (parameterize_ac_effects(bterms)) {
```

Comment:
To make sure everything is in order with the code below, I would need to discuss this code in a meeting I think.